src/components/GraphWrapper/graphWrapper.test.tsx

```diff
@@ -12,11 +12,8 @@ import { queryEntities } from "../../services/queryEntity";
 import constants from "../../utils/constants";
 import { useGlobalStore } from "../../store/global";
 
-const {
-  defaultStartTimestamp,
-  defaultStepValue,
-  defaultEndTimestamp,
-} = constants;
+const { defaultStartTimestamp, defaultStepValue, defaultEndTimestamp } =
```

Comment:
shouldn't be a part of the changes, please enable prettier on your system to auto save formattings